webapp/views/wpt-results.js

```diff
@@ -829,10 +850,16 @@ class WPTResults extends WPTColors(WPTFlags(PathInfo(LoadingState(TestRunsUIBase
     }
   }
 
-  platformID({browser_name, browser_version, os_name, os_version}) {
+  platformID({ browser_name, browser_version, os_name, os_version }) {
     return `${browser_name}-${browser_version}-${os_name}-${os_version}`;
   }
 
+  hasAmendableMetadata(node, index, testRun) {
+    const totalTests = this.getNodeResultDataByPropertyName(node, index, testRun, 'total');
+    const passedTests = this.getNodeResultDataByPropertyName(node, index, testRun, 'passes');
+    return this.computePathIsATestFile(node.path) && (totalTests - passedTests) > 0;
```

Comment:
Here's a case where this doesn't work as expected (the test is MISSING, so the totals are 0/0)
https://wpt.fyi/results/FileAPI?label=master&product=chrome&product=firefox&product=safari&q=%28chrome%3A%21pass%26chrome%3A%21ok%29+%28firefox%3Apass%7Cfirefox%3Aok%29+%28safari%3Apass%7Csafari%3Aok%29

Reply:
When it says MISSING, it's supposed to mean that the test result is missing right? Maybe it should show 0/1 instead of 0/0?

Reply:
the summary is [passes] / [total], and the total number of executed tests can vary (it shouldn't, but it can). We don't know how many tests "should" run for a result that's present in some other product's run, and we can't necessarily trust that run's total.